indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -281,11 +281,12 @@
 
       var self = this,
           renderCoverpage = this.parent.fragment.parentElement === null,
-          $akn = this.$('.document-workspace-content .akoma-ntoso'),
+          $akn = this.$('.document-workspace-content la-akoma-ntoso'),
           coverpage;
 
       // reset class name to ensure only one country class
-      $akn[0].className = "akoma-ntoso spinner-when-empty country-" + this.parent.model.get('country');
+      $akn[0].className = "akoma-ntoso spinner-when-empty";
+      $akn[0].frbrCountry = this.parent.model.get('country');
```

Comment:
This can fall away when the expression frbr uri is set in the HTML.

Reply:
done